js/jquery.mapael.js

```diff
@@ -290,9 +290,16 @@
                             delete plots[deletedPlots[i]];
                         }
                     }
+                // Delete ALL plots if deletedPlots is set to "all"
+                } else if (deletedPlots === "all") {
+                    $.each(plots, function(id, elem) {
+                        fnRemoveElement(elem);
+                    });
+                    // Empty plots array
+                    plots.length = 0;
```

Comment:
Hmm, in any case, I think it shouldn't work properly because the variables plots and areas are objects and not arrays, so they shouldn't have the 'length' property.

Moreover, by using this way for resetting an array, I wonder how the garbage collector behaves ?